x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.scss

```diff
@@ -9,6 +9,13 @@
   overflow: visible;
   height: 100%;
 
+  .lnsWorkspacePanelWrapper__content {
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
```

Comment:
why do you need to hide overflow-x?

Reply:
We don't! I just did many tests and I forgot to clear this. Thanx! 
[Fixed](https://github.com/elastic/kibana/pull/143340/commits/79c429137691972bdcb2803be1e4a4ed1bc65284)